twitter_acquier.rb

```diff
@@ -1,47 +1,25 @@
-class TwitterAcquier
-  def session_to_google_auth
-    credentials = Google::Auth::UserRefreshCredentials.new(
-      client_id: ENV['GOOGLE_CLIENT_ID'],
-      client_secret: ENV['GOOGLE_CLIENT_SECRET'],
-      scope: [
-        "https://www.googleapis.com/auth/drive",
-        "https://spreadsheets.google.com/feeds/",
-      ],
-      refresh_token: ENV['GOOGLE_REFRESH_TOKEN'])
-    session = GoogleDrive::Session.from_credentials(credentials)
-    session
+class TwitterScraper
+  def login_to_twitter(agent, id, pass)
+    page = agent.get('https://twitter.com/')
+    form = page.forms[1]
+    form.field_with(name: "session[username_or_email]").value = id
+    form.field_with(name: "session[password]").value = pass
+    form.submit
+    puts 'logged_in_twitter'
   end
 
-  def find_work_sheet(session, sp_url)
-    sp = session.spreadsheet_by_url(sp_url)
-    ws = sp.worksheet_by_title("test_volley")
-    ws
-  end
-
-  def send_data_to_sp(analytics_file, ws)
-    i = 1
-    CSV.parse(analytics_file).each do |csv|
-      a = 1
-      for a in 1..40 do
-        ws[i,a] =csv[a-1]
-      end
-      i += 1
-    end
-    puts 'success' if ws.save
-  end
-
-  def sending_url(id)
+  def set_scraping_url(id)
     start_date = (Time.now.beginning_of_month).to_i.to_s + "000"
     end_date = (Time.now).to_i.to_s + "999"
```

Comment:
![2018-08-31 15 52 49](https://user-images.githubusercontent.com/35832220/44897908-46a82a80-ad38-11e8-8b84-bdec68aca22b.png)

I want to obtain data from beginning of the month to now.
(So now 8/1~8/31)

But in fact, process obtains data from 7/31~ now.
https://docs.google.com/spreadsheets/d/1mNAJZp0B0nLZ6dYuoMZuYDmIRUd_w8PiiY_NtEUFTW8/edit#gid=1823441032
I don't know the reason.


[Q] Do you the reason why? @ladnack @nafu 

Tomorrow is the first of month, so I tried checking.
In July, I can get (7/23~7/31) because business team only tweet in this period. So I can't check.

Reply:
Thank you for reply.

>1.Check your request is what you intend. If not, you need to fix to compose your request
 
request is fine.
![2018-08-31 16 23 48](https://user-images.githubusercontent.com/35832220/44898574-49a41a80-ad3a-11e8-8d9b-40a320d210e4.png)


>2.Check the response is what you intend. If not, you need to know more details of Twitter's specification

Response is not what I intend.
So I should search.

Reply:
https://analytics.twitter.com/user/playerapp_vb/tweets/export.json?start_time=1533081600000&end_time=1535759999999&lang=ja